src/kubernetes-ingress-controller/concepts/design.md

```diff
@@ -59,6 +63,39 @@ configuration:
   This means that all requests flowing through Kong are not directed via
   kube-proxy but directly to the pod.
 
-For more information on how Kong works with Routes, Services, and Upstreams,
-please see the [Proxy](/gateway/latest/reference/proxy/)
-and [Load balancing](/gateway/latest/how-kong-works/load-balancing/) references.
+[k8s-service]: https://kubernetes.io/docs/concepts/services-networking/service/
+
+### Ingress
+
+An [Ingress][ingress] resource in Kubernetes defines a set of rules for proxying
+traffic. These rules corresponds to the concept of Route in Kong.
+
+The following figure describes the mapping between Kubernetes concepts and Kong's
+configuration, while using `Ingress`:
+
+![translating Kubernetes to Kong](/assets/images/docs/kubernetes-ingress-controller/k8s-to-kong.png "Translating k8s resources to Kong")
+
+### Gateway API
+
+Gateway API resources can also be used to produce running instances
+and configurations for {{site.base_gateway}}.
+
+The main concepts here are:
+
+- A [`Gateway`][gateway-api-gateway] resource in Kubernetes describes how traffic
+  can be translated to Services within the cluster.
+- A [`GatewayClass`][gateway-api-gatewayclass] defines a set of Gateways that share
+  a common configuration and behaviour.
+  Each `GatewayClass` will be handled by a single controller, although controllers
+  may handle more than one `GatewayClass`.
+- [`HTTPRoute`][gateway-api-httproute] can be attached to a Gateway which will
+  configure the HTTP routing behavior.
+  
+You can find more details about Gateway API concepts supported by {{site.kic_product_name}}
+in [here](/kubernetes-ingress-controller/latest/references/gateway-api-support).
```

Comment:
I think `here` is better than `in here` here, in grammar.

Reply:
I'll leave that up to the editors of docs. Thanks for bringing this up!